spec.bs

```diff
@@ -245,6 +245,12 @@ spec: css2; urlPrefix: https://www.w3.org/TR/CSS21/visuren.html
   type: dfn
     for: css2
       text: viewport; url: viewport
+spec: attribution-reporting; urlPrefix: https://wicg.github.io/attribution-reporting-api/
+  type: dfn
+    for: eligibility
+      text: event-source; url:eligibility-event-source
+      text: navigation-source; url:eligibility-navigation-source
+      text: unset; url:eligibility-unset
```

Comment:
nit: in all of these, let's add a space after the colon after `url:`, to match the other links here.